docs/internal-developers/testing/releases/1001.md

```diff
@@ -0,0 +1,35 @@
+# Testing notes and ZIP for release 10.0.1
+
+Zip file for testing: [woocommerce-gutenberg-products-block.zip](https://github.com/woocommerce/woocommerce-blocks/files/11260936/woocommerce-gutenberg-products-block.zip)
+
+## WooCommerce Core
+
+### Single Product Compatibility Layer: add support for custom HTML Blocks. [(9075)](https://github.com/woocommerce/woocommerce-blocks/pull/9075)
+
+1. Go to Appearance > Editor and in the Single Product template before upgrading to the blockified version, add a HTML Block _outside_ of the Group block as shown in the screenshot below with the contents `<h2>HTML Block here</h2>` (or similar)
+2. Save and view this block on the frontend.
+3. Return to the template, upgrade it to the blockified version
+4. Save and view on the frontend. Be sure that the HTML is visible on the page.
+
+![Screenshot 2023-04-17 at 14 34 13](https://user-images.githubusercontent.com/8639742/232499923-9ca7cb7a-c4e7-417d-af78-a8b86b87ea10.png)
+
+### Blockfied Single Product Template: Add support for template for specific product. [(9069)](https://github.com/woocommerce/woocommerce-blocks/pull/9069)
+
+1. Go to Editor > Manage Templates > Add new > Single Post: Product > Select a product with multiples images
```

Comment:
For me, this is menu element is called `Single Item: Product`:

![imatge](https://user-images.githubusercontent.com/3616980/232767595-b73501d9-a018-49cc-8e11-703277962e97.png)

I would maybe rephrase the entire sentence just in case:

```suggestion
1. Go to Appearance > Editor > Templates > Add new (plus icon) > Single Item: Product > Select a product with multiples images
```